### PR TITLE
Add old way to check unapplied migration

### PR DESCRIPTION
**Pull request by tyzhnenko**

### CODE REVIEW

**Comment by ecugol** [comment]
Hello @tyzhnenko. This is a nice addition, but i think it should be named differently, because check-unapplied-migrations supposed to check if we have migration file which is not applied on a DB, while `makemigrations --check` will return non-zero when we have changes in the models but we don't have a migration file for a such change.

File: README.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ Check for unapplied migrations with manage.py migrate --check
 
 #### `check-absent-migrations-old`
```
Comment (ecugol) [inline README.md:47]: @tyzhnenko This should be renamed too

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -11,6 +11,12 @@
     pass_filenames: false
     language: system
     always_run: true
+-   id: check-absent-migrations
+    name: Check unapplied migrations with manage.py makemigrations --check
```
Comment (ecugol) [inline .pre-commit-hooks.yaml:15]: @tyzhnenko name should be renamed according to the logic

File: README.md [hunk 3]
```diff
@@ -18,6 +18,7 @@ Add this to your `.pre-commit-config.yaml`
         # otherwise it will work on all branches
         args: ["--branches", "main", "other_branch"]
     -   id: check-unapplied-migrations
+    -   id: check-unapplied-migrations-old
```
Comment (ecugol) [inline README.md:21]: @tyzhnenko please rename the id

File: README.md [hunk 4]
```diff
@@ -43,6 +44,10 @@ Forbids commit if untracked migrations files are found (e.g. `*/migrations/0001_
 
 Check for unapplied migrations with manage.py migrate --check
 
+#### `check-absent-migrations-old`
```
Comment (ecugol) [inline README.md:47]: @tyzhnenko needs to be renamed as well

**Comment by tyzhnenko** [comment]
lgmt and make a new release :)

**Comment by ecugol** [comment]
@tyzhnenko thanks for the contribution. I'll release this soon.

**Pull request merged**